qa/L0_java_memory_growth/test.sh

```diff
@@ -83,13 +85,15 @@ CLIENT_LOG="./client_$LOG_IDX.log"
 
 # Longer-running memory growth test 
 ITERS=1000000
+MAX_MEM_GROWTH=10
 if [ "$TRITON_PERF_LONG" == 1 ]; then
     # ~1 day
     ITERS=125000000
+    MAX_MEM_GROWTH=25
 fi
 
 echo -e "\nRunning Memory Growth Test, $ITERS Iterations\n"
-$BASE_COMMAND -Dexec.args="-r $MODEL_REPO -c -i $ITERS" >>$CLIENT_LOG 2>&1
+$BASE_COMMAND -Dexec.args="-r $MODEL_REPO -c -i $ITERS --max-growth $MAX_MEM_GROWTH" >>$CLIENT_LOG 2>&1
```

Comment:
Can we set max heap size? I think this could help with the machine dependent growth behavior

Reply:
I don't believe so. The result is simply the program crashing (out of memory error), not the memory being managed differently.

This was what happened last time, but I tested it again to make sure. When I set it to 64MB, it crashed due to the memory running out during runtime. When I set it to 512MB, the behavior did not change. If you'd prefer, I can set the memory higher, but it doesn't seem to change the memory management.

Reply:
Tested the other garbage collection options and parameters you sent over. Ultimately, ended up setting XX:MaxGCPauseMillis to 40ms instead of the default of 200ms. The garbage collector should be collecting more frequently since it's aiming to pause for a shorter window each time.

This seems to be resulting in slightly less memory growth during test runs, but we'll see in time if that consistency continues in the long-running stress test.